### ISSUE TITLE
Staging Review finding: Image missing alt text

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#125855]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV-Secure Messaging
**Product:** Secure Messaging
**Feature:** Medical Renewal Via Secure Messaging

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Original fill image is an image with a lot of text content, however that content is not provided to screen reader users. The text below the image is "Medication description. No description available. Call your VA pharmacy at 783-272-1430 if you need help identifying this medication." and I assume this text is supposed to match the medication label text in the image, however in this scenario, content and context is being provided to sighted users, but screen reader users are not provided an equivalent experience. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="832" height="538" alt="no alt text meds" src="https://github.com/user-attachments/assets/8dc037e6-e9f1-40bb-93f8-2146d70b5234" />

</details>

## Recommended action

Ensure that the text below the image captures the intent and context of the medication refill image, or provide alt text and potentially additional context through use of a `<figure><figcaption>` to ensure screen reader users are provided the same information and an equivalent experience. 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG22/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by michaelbrodsky-a6** [comment]
Hey @sarahhortonadhoc – just wanted to briefly check in on this. I know that Ian had created ticket [#40330](https://github.com/department-of-veterans-affairs/vets-website/issues/40330) to mirror this (like you've outlined above), and we just refined that ticket during Wednesday's refinement ceremony. But I just noticed that Raul is assigned to this version of the ticket. Do you know if he's already working on this or if it's still a to-do for the medications team?

**Comment by rretamoza-va** [comment]
@michaelbrodsky-a6 Since Medications is working a ticket mirroring this, should I assign this one to you?

**Comment by michaelbrodsky-a6** [comment]
No problem at all @sarahhortonadhoc! And thanks so much @rretamoza-va – just seeing your note. I think we can actually go ahead and mark this ticket as "Close as not planned" now, and unassign you from it, since the meds has undertaken the work in this mirror ticket instead ([#40330](https://github.com/department-of-veterans-affairs/vets-website/issues/40330)). Really appreciate your follow-up on this front.
